cfgov/v1/tests/wagtail_pages/helpers.py

```diff
@@ -0,0 +1,19 @@
+from v1.models.home_page import HomePage
+from django.contrib.auth.models import User
+
+
+def save_page(page):
+	admin_user = User.objects.get(username='admin')
+	page.save()
```

Comment:
Any test that publishes a page ends up altering the database state for other tests. How would you feel about adding `tearDown` methods to the tests that cleanup the changes they make, so as to keep tests independent?